src/gentropy/ukb_ppp_eur_sumstat_preprocess.py

```diff
@@ -38,11 +49,17 @@ def __init__(
                 raw_study_index_path_from_tsv=raw_study_index_path_from_tsv,
                 raw_summary_stats_path=raw_summary_stats_path,
             )
-            .df
-            .write
-            .mode("overwrite")
+            .df.withColumn("biosampleFromSourceId", f.lit("UBERON_0001969"))
```

Comment:
I know I will be annoying, but could you move this line to the from_source method for consistency.  So we have the full description of study_index there?

Reply:
Done